docs/docs/tutorials/gettingstarted.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Getting Started with Springboard
+summary: A walkthrough of how to pull down the Springboard project, get it running for the first time in Intellij and 
+start customizing a few pieces of the project.
```

Comment:
Ah! I think the newline in here is breaking the change:

https://happyvalleyio.github.io/springboard/tutorials/gettingstarted/

Reply:
I have fixed this!